backend/uclapi/oauth/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from .app_helpers import generate_user_token
+
+class OAuthScope(models.Model):
+    # We should really have a primary key for the OneToOneField
+    id = models.AutoField(primary_key=True)
```

Comment:
Oh, good to know! I like having the verbosity of it though; I feel more comfortable actually seeing my PK :P